src/faust/rack.dsp

```diff
@@ -28,14 +28,14 @@
 //
 // The conversion formula is: 440 * 2 ^ (volts - 0.75)
 // The factor 0.75 shifts 0 V to C-4 (261.6256 Hz)
-cv_pitch2freq(cv_pitch) = cv_pitch , 0.75 : 2 , - : 440 , ^ : * : _;
+cv_pitch2freq2(cv_pitch) = 440 * 2 ^ (cv_pitch - 0.75);
```

Comment:
Is the trailing 2 in `cv_pitch2freq2` a typo or some sort of convention? Otherwise I think it should be removed for consistency.

Reply:
Typo ! (I did some tests in another file!), just remove it.